estate/models/estate_model.py

```diff
@@ -0,0 +1,33 @@
+from odoo import fields, models
+
```

Comment:
You should have 2 empty lines before a class definition according to 'PEP 8: E302 expected 2 blank lines, found 1'